rest_framework_gis/filters.py

```diff
@@ -188,3 +189,21 @@ def filter_queryset(self, request, queryset, view):
             dist = self.dist_to_deg(dist, point[1])
 
         return queryset.filter(Q(**{'%s__%s' % (filter_field, geoDjango_filter): (point, dist)}))
+
+
+class DistanceToPointOrderingFilter(DistanceToPointFilter):
+    srid = 4326
+
+    def filter_queryset(self, request, queryset, view):
+        from django.contrib.gis.db.models.functions import GeometryDistance
```

Comment:
this import here is not great, maybe we can import it at the top and catch the exception?

Something like 
```python
try:
    from django.contrib.gis.db.models.functions import GeometryDistance
except ImportError:
    GeometryDistance = False
```

and in this method do something like:

```python
if not GeometryDistance:
    raise ValueError('GeometryDistance not available on this version of django')
```